web/gds-user-ui/src/modules/dashboard/member/components/MemberModal/Copy.tsx

```diff
@@ -0,0 +1,110 @@
+import { useEffect, useState } from "react";
+import { copyToClipboard } from "../../utils";
+import { Button } from "@chakra-ui/react";
+import { Trans, t } from "@lingui/macro";
+import { getBusinessCategoryValue, getVaspCategoryValue } from "constants/basic-details";
+
+interface MemberDetailTableHeader {
+  label: string;
+  value: string;
+}
+
+type CopyProps = {
+    data: any;
+};
+
+function Copy({ data }: CopyProps) {
+const memberDetailTableHeader = [
+  {
+    label: t`Name`,
+    value: data.summary.name
+  },
+  {
+    label: t`Website`,
+    value: data.summary.website
+  },
+  {
+    label: t`Business Category`,
+    value: getBusinessCategoryValue(data.summary.business_category)
+  },
+  {
+    label: t`VASP Category`,
+    value: getVaspCategoryValue(data.summary.vasp_categories)
+  },
+  {
+    label: t`Country of Registration`,
+    value: data.legal_person.country_of_registration
+  },
+  {
+    label: t`Technical Contact Name`,
+    value: data.contacts.technical.name
+  },
+  {
+    label: t`Technical Contact Email`,
+    value: data.contacts.technical.email
+  },
+  {
+    label: t`Technical Contact Phone`,
+    value: data.contacts.technical.phone
+  },
+  {
+    label: t`Compliance / Legal Contact Name`,
+    value: data.contacts.legal.email
+  },
+  {
+    label: t`Compliance / Legal Contact Email`,
+    value: data.contacts.legal.email
+  },
+  {
+    label: t`Compliance / Legal Contact Phone`,
+    value: data.contacts.legal.phone
+  },
+  {
+    label: t`Administrative Contact Name`,
+    value: data.contacts.administrative.name
+  },
+  {
+    label: t`Administrative Contact Email`,
+    value: data.contacts.administrative.email
+  },
+  {
+    label: t`Administrative Contact Phone`,
```

Comment:
Can null safety be implemented using optional chaining? If the data is unavailable, can "N/A" be used as a fallback value?

Reply:
Sure, I'll added the null safety. 

`N/A` is included in `copyToClipboard`:

```
export async function copyToClipboard(data: any) {
  try {
    const values = data.map((item: any) => {
      item.value = item.value ? item.value : 'N/A';
      return ` ${item.label} \n ${item.value} \n`;
    }).join('\n');
 ```